src/web/app/pages-admin/admin-home-page/admin-home-page.component.ts

```diff
@@ -121,10 +124,19 @@ export class AdminHomePageComponent {
           instructor.statusCode = 200;
           instructor.joinLink = resp.joinLink;
           this.activeRequests -= 1;
-        }, (resp: ErrorMessageOutput) => {
+        }, (resp: ErrorMessageOutput | AccountRequestCreateErrorResultsWrapper) => {
           instructor.status = 'FAIL';
           instructor.statusCode = resp.status;
-          instructor.message = resp.error.message;
+          if ('message' in resp.error) {
+            // resp is ErrorMessageOutput
+            instructor.message = resp.error.message;
+          } else {
+            // resp is AccountRequestCreateErrorResultsWrapper
+            instructor.message = [resp.error.invalidNameMessage, resp.error.invalidEmailMessage,
+              resp.error.invalidInstituteMessage]
+              .filter(Boolean)
+              .join(' ');
```

Comment:
No need to change the behaviour of this page, not in scope.

Reply:
The return type of errors from backend is changed. That's why this handling of error needs changes.

Reply:
This change has been reverted and no longer needed after changing `CreateAccountRequestAction`. The new error type is not applicaple for `ADMIN_CREATE`.